README.md

```diff
@@ -1,16 +1,16 @@
 # BeLL apps Angular reboot prototype **Planet**
 
-Project to create a prototype for a reboot of the BeLL apps using Angular4 & CouchDB.
+Project to create a prototype for a reboot of the BeLL apps using Angular5 & CouchDB2.
 
 ## To work on this
 
-The only prequisite is Vagrant.  After cloning the repository, run `vagrant up` in the console.  Once it's done installing the virtual machine it'll automatically start compiling the app.  After about 10 seconds, you can open the app at `localhost:3000`
+The only prerequisite is Vagrant. If you don't know about it, please do some research and try it. After cloning the repository, run `vagrant up` in the console. Once it's done installing the virtual machine it'll automatically start compiling the app.  After about 10 seconds, you can open the app at `localhost:3000`.

```

Comment:
NOTE: CPU or BIOS should have virtulization (VT-x) enabled, Parallelization is not enough.
Also for windows Pro Hypervisor should not be enabled

Reply:
thanks @mappuji and @lmmrssa but since non of this changes in `README.md` have nothing to do with the feature of this PR please ignore that for right now

Reply:
gettext